flow-server/src/test/java/com/vaadin/flow/server/frontend/AbstractUpdateImportsTest.java

```diff
@@ -87,6 +87,12 @@ public static class FooCssImport extends Component {
 
     }
 
+    @Route(value = "themeform")
```

Comment:
maybe a super minor typo?
```suggestion
    @Route(value = "themefor")
```
Not sure, `themeform` also makes sense, but do not see a form here :)